website/components/CodeDocs.tsx

```diff
@@ -156,7 +157,7 @@ function DesktopInstall({ component, changelog, version }: InstallProps) {
 
 function CodeDocs({ component, readme, changelog }: BaseLayoutProps) {
   const viewport = useViewportSize();
-  const isMobile = viewport?.width < breakpoints.Tablet;
+  const isMobile = viewport?.width! < breakpoints.Tablet;
```

Comment:
comparing non-numbers is sketchy*. we should explicitly specify `true` or `false` when `viewport` is not defined.

\*
```shell
$ null < 1
true

$ undefined < 1
false
```

​

Reply:
👍